docs/source/tensors.rst

```diff
@@ -315,6 +315,7 @@ view of a storage and defines numeric operations on it.
    .. automethod:: fliplr
    .. automethod:: flipud
    .. automethod:: float
+   .. automethod:: float_power
    .. automethod:: floor
```

Comment:
and `float_power_` once inplace is added (per above)

Reply:
Addressed!